### DISCUSSION TITLE
How to drop the column name used for groupkfold cross validation while creating the model?

### DISCUSSION THREAD

**Discussion by srinivas2036** [pycaret/pycaret#2761] [Q&A]
I am working on my research project, where I need to develop a small model for my work. I am using pycaret for that. In pycaret if I am using 'groupkfold', I need to pass a string representing the column name for that. However, the data in that column if I don't what to use how to do that. 

from pycaret.regression import *
from pycaret.datasets import get_data
data = get_data('diamond')
reg2 = setup(data, target = 'Price', session_id=786,
             transform_target = True,
             bin_numeric_features=['Carat Weight'],
             remove_multicollinearity=True, ignore_features=['Color'],fold_groups='Color',fold_strategy='groupkfold',
             feature_interaction=True)
lr=create_model('lr',fold=3)

is this the right way of doing. I am not able to understand whether folds are generated based on column -'Color'? If they are generated based on color column, is there a way to return which attributes of color are present in each fold?
Kindly respond. Thank you for the time.

**Comment by Ruibin-Liu** [comment]
Hello, have you figured this out? I also want to use a column to label the groups but not to be included in features for training. Using the code similar to yours doesn't group at all.
